.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: haskell
+
+ghc:
```

Comment:
I just put everything for now, see what works. No one has ever said what we do and don't support.